code/game/turfs/simulated/walls_misc.dm

```diff
@@ -26,6 +26,15 @@
 	icon = 'icons/turf/walls.dmi'
 	icon_state = "rockvault"
 
+/turf/simulated/wall/ice
+	icon = 'icons/turf/walls/icedmetal_wall.dmi'
+	icon_state = "iced"
+	desc = "A wall covered in a thick sheet of ice."
+	walltype = "iced"
+	canSmoothWith = list(/turf/simulated/wall/ice)

```

Comment:
I dunno man, I was literally just doing a local test without it and they only smoothed with the regular walls and not themselves.